Build/Tactical/Drugs_And_Alcohol.cc

```diff
@@ -243,8 +243,8 @@ void HandleEndTurnDrugAdjustments( SOLDIERTYPE *pSoldier )
 			{
 				pSoldier->bDrugEffect[ cnt ] = 0;
 
-        // Dirty panel
-        fInterfacePanelDirty = DIRTYLEVEL2;
+                                // Dirty panel
+                                DirtyMercPanelInterface(pSoldier, DIRTYLEVEL2);                                
```

Comment:
[3rd commit](https://github.com/ja2-stracciatella/ja2-stracciatella/pull/403/commits/decfafd6dc7d576b9f166317dd2ab51d8e1f3334) fixed that.